packages/common/src/system/system.spec.ts

```diff
@@ -107,25 +107,31 @@ describe('serializeError()', () => {
 });
 
 describe('normalizeHttpResponse()', () => {
-    it('should normalize request object', () => {
+    it('should normalize HTTP request object', () => {
+        const requestObj = {
+            url: 'url',
+            method: 'PUT',
+            headers: [{ name: 'Accept', value: 'application/xml' }],
+            body: 'body',
+            operationSpec: { path: '/{containerName}/{blob}' },
+        } as any;
         const httpResponse = {
+            name: 'RestError',
+            code: 'ConditionNotMet',
             statusCode: 412,
-            request: {
-                url: 'request url',
-                method: 'PUT',
-                body: 'request body',
-            },
+            request: requestObj,
             response: {
-                body: 'response body',
+                headers: [{ name: 'content-length', value: '255' }],
+                request: requestObj,
+                status: 412,
+                bodyAsText: 'bodyAsText',
             },
         } as any;
-        const { request, ...httpResponseExpected } = httpResponse;
-        httpResponseExpected.request = {
-            url: 'request url',
-            method: 'PUT',
-        };
+        const expectedResponse = JSON.parse(
```

Comment:
Why is JSON.parse called here instead of creating the object?

Reply:
to simplify test code, similar to snapshot